lib/options.js

```diff
@@ -540,9 +602,12 @@ ${Flags.buildFlag(subcommand, Flags.gitHubAuthFlag)}`
   return {gitHubUser, gitHubPassword};
 }
 
+/**
+ *
+ * @param {ErrorMessage.CustomError} errorToReport
```

Comment:
I think you're missing the import for `CustomError`, I don't think this is in scope.